### ISSUE TITLE
Data channel support missing from stable?

### ISSUE THREAD

**Issue by jexe** [RSATom/janus-gateway-snap#9]
Hi,
Thanks for this snap. I've been using it on edge, but wanted to switch to latest/stable for more reliability - however, I'm running into this issue when trying to use textroom on stable:

`[WARN] Data channels support not compiled, disabling TextRoom plugin`

Is this an intentional build difference between edge and stable?

**Comment by RSATom** [comment]
Yes, there is no data channels support in any stable snap channel, just because data channels support in Janus is based on `usrsctp` library, and this library just don't define any stable version.

**Comment by kylebakerio** [comment]
This might be a deal killer. Is there any other option, like to make two snaps available, one with and one without? Or is it available in a `testing` snap channel? (New to snap, sorry if this is a silly question).

**Comment by RSATom** [comment]
@kylebakerio by default snap install installs from `stable` channel (and there is not datachannels support there by mentioned above reason). So to get datachannels support you have to use `edge` channel, so use `sudo snap install janus-gateway --edge`
